src/parse_nlp.jl

```
@@ -11,6 +11,10 @@ function _let_code_block(ex::Expr)
     return ex.args[2]
 end
 
+function _error_curly(x)
```

Why is this in `parse_nlp`? We should also throw this error for the regular macros.

---

It is now called inside MutableArithmetics for the regular macro.